package.json

```diff
@@ -7,10 +7,12 @@
     "test": "tests"
   },
   "scripts": {
+    "css:build": "node-sass --omit-source-map-url ./public/style/mystyles.scss ./public/style/style.css",
+    "css:watch": "npm run css:build -- --watch",
     "test": "jest --config=jest.config.js -i --forceExit",
     "test:dev": "jest --config=jest.config.js -i --watchAll",
     "start": "npm run server",
-    "start:dev": "npm run build:dev & npm run server:dev",
+    "start:dev": "npm run css:watch & npm run build:dev & npm run server:dev",
     "build": "./node_modules/.bin/webpack  --config webpack.production.config.js",
```

Comment:
I think you might have to add `& npm run css:build` to the build script for the compiled scss file to be produced in production?

Reply:
yeah this is the stuff I was unsure about, I am sure you are correct. I was worried about these changes not showing up in production, but unsure which scripts to modify in order to make that happen. Thanks a lot I'll definitely add it

Reply:
The git-ignore issue I am honestly unsure about - this is my first time using a git-ignore file and also using node-sass, I will look into it but if you have any opinion on this I'm open to going with it

Reply:
I am pretty sure that edit to the build script is required. So build would be `./node_modules/.bin/webpack  --config webpack.production.config.js & npm run css:build` where css:build is `path/to/binary/in/node_modules --omit-source-map-url ./public/style/mystyles.scss ./public/style/style.css`

Reply:
That way the scss will be compiled and produce the file.

Reply:
As for git ignoring, I'm pretty sure you should add `/public/style.css` to gitignore b/c it is the produced file, and we only need to track the file that produces it.

Reply:
yeah all of this makes sense, thanks a lot, i'll push up those changes in a few minutes

Reply:
Just to clarify, when adding the npm run css:build to the build script, should I change css:build to:

path/to/binary/in/node_modules --omit-source-map-url ./public/style/mystyles.scss ./public/style/style.css

I don't need to include anything about node-sass in the new css:build code?

Reply:
I am unable to push up any changes, github is giving me a really hard time and errors I've never seen before